docs/pages/enterprise/sso.mdx

```diff
@@ -237,5 +237,88 @@ must be able to:
 If something is not working, we recommend to:
 
 - Double-check the host names, tokens and TCP ports in a connector definition.
-- Look into Teleport's audit log for claim mapping problems. It is usually stored on the
-  auth server in the `/var/lib/teleport/log` directory.ad
+- Review the below log information and the troubleshooting sections for the above guides.
+
+<Tabs>
+<TabItem label="Self-Hosted" scope={["oss","enterprise"]}>
+
+### Using the Web UI
+
+If you get "access denied" or other login errors, the number one place to check is the Audit
+Log on the Teleport Auth Server. You can access it in the **Activity** tab of the Teleport Web UI.
+
+![Audit Log Entry for SSO Login error](../../img/sso/teleportauditlogssofailed.png)
+
+Example of a user being denied because the role `clusteradmin` wasn't set up:
+
+```json
+{
+  "code": "T1001W",
+  "error": "role clusteradmin is not found",
+  "event": "user.login",
+  "method": "oidc",
+  "success": false,
+  "time": "2019-06-15T19:38:07Z",
+  "uid": "cd9e45d0-b68c-43c3-87cf-73c4e0ec37e9"
+}
+```
+
+### On the Auth Service host
```

Comment:
I don't know if we should even mention this. The web UI approach works in all cases, and this approach works only in a small subset of cases.

Reply:
removed